docs/src/styles/docs/header.scss

```diff
@@ -67,10 +74,12 @@
     -1 * var(--amplify-components-button-small-padding-inline-start)
   );
 
-  @media (min-width: $breakpoint-large) {
-    display: none;
-    .docs-home & {
-      display: inline-flex;
+  &:not(.always-collapsible) {
+    @media (min-width: $breakpoint-large) {
+      display: none;
+      .docs-home & {
```

Comment:
Can we change the homepage to use the `alwaysCollapsible` context so we don't have to have this specific style?